core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -87,6 +91,34 @@ describe('Topics and skills dashboard functionality', function() {
     topicEditorPage.expectNumberOfUncategorizedSkillsToBe(1);
   });
 
+  it('should merge an outside skill with one in a topic', function() {
+    topicsAndSkillsDashboardPage.createSkillWithDescription(
+      'Skill to be merged');
+    skillEditorPage.editConceptCard('Concept card explanation');
+    skillEditorPage.saveOrPublishSkill('Added review material.');
+    skillEditorPage.moveToQuestionsTab();
+    skillEditorPage.clickCreateQuestionButton();
+    explorationEditorMainTab.setContent(forms.toRichText('Question 1'));
```

Comment:
I don't think this code is readable! we are editing the question using `explorationEditorMainTab`.

Reply:
Yes, because we are using the same directives (for state editor) as used in explorations for questions, so it didn't make sense to duplicate code for question editor tests.

Reply:
@aks681, Can we have a stateEditor in protarctor_utils which we can use in both the explorationEditor tab and in questionEditor page?

Reply:
Can we file an issue for this refactor and do it after this PR? I think that's more of a conceptual change, that shouldn't block the release. What do you think?

Reply:
I would recommend that this doesn't block the release, yep.

Reply:
I was too not sure about adding this in this PR, @seanlip, and @nithusha21 Thanks for the suggestion.
I agree that we can do this in another PR (taking release in consideration). 

I've created an issue for the same at #6509.